packages/components/select-v2/src/useSelect.ts

```diff
@@ -801,9 +801,11 @@ const useSelect = (props: ISelectV2Props, emit: SelectEmitFn) => {
   watch(
     () => props.modelValue,
     (val, oldVal) => {
+      const isValEmpty = !val || (Array.isArray(val) && val.length === 0)
```

Comment:
Use the internal packaged function.
```suggestion
      const isValEmpty = !val || (isArray(val) && val.length === 0)
```